config_models/models.py

```diff
@@ -5,25 +5,18 @@
 
 from django.db import models
 from django.conf import settings
-from django.core.cache import caches, InvalidCacheBackendError
 from django.utils.translation import ugettext_lazy as _
 from django.utils import six
 
+from edx_django_utils.cache.utils import TieredCache
 from rest_framework.utils import model_meta
-# Config model values are cached. The caching backend configuration used has implications for how the values
-# are changed/queried in an active release. When a new key/value is saved, its current cached value is cleared
-# via deletion from the cache. Otherwise, depending on the cache backend configuration *and* on the value of
-# ConfigurationModel.cache_timeout, the key/value may still exist in some caches.
-try:
-    # Use the Django settings-configured cache, which could be:
-    #  - memcache or another external networked cache (per-cluster)
-    #  - file cache (per-machine)
-    #  - local memory cache (per-process)
-    #  - other
-    cache = caches['configuration']  # pylint: disable=invalid-name
-except InvalidCacheBackendError:
-    # If no caches configured, use the default cache specified in Django CACHES setting.
-    from django.core.cache import cache
+
+# The following import exists for backwards compatibility (because a number of
+# library users assume config_models.models.cache is importable), but
+# ConfigModels will now ignore the cutom 'configuration' cache setting and just
+# use TieredCache, which will make use of a local request cache + the default
+# Django cache.
+from django.core.cache import cache  # pylint: disable=unused-import
```

Comment:
Yeah, maybe I should just fix the references in edx-platform. It's used because we did some custom extension to make course-specific waffle flag checks use a request cache (before TieredCache was available I think). But I can search/replace for those things–I was just being lazy as this is my off-hours work.